bbc_microbit build failure is blocking CI

The `bluetooth.mesh.main.microbit` case in tests/bluetooth/mesh is failing and causing CI failures in other PRs.

I'm getting this error in CI in an unrelated PR (#31965):

```
/workdir/zephyr/subsys/bluetooth/controller/ll_sw/ull.c:106:7: error: 'BT_CTLR_CONN_MAX' undeclared here (not in a function)
  106 |       BT_CTLR_CONN_MAX + \
      |       ^~~~~~~~~~~~~~~~
```

Details in https://buildkite.com/zephyr/zephyr/builds/21988#868a8486-8a62-4f2a-9274-f55581716664.

This reproduces for me on master with:

```
west build -p -b bbc_microbit -s tests/bluetooth/mesh -- -DCONF_FILE=$PWD/tests/bluetooth/mesh/microbit.conf
```

I'm using Zephyr SDK 0.12.2.

--- a/subsys/bluetooth/controller/ll_sw/ull.c
+++ b/subsys/bluetooth/controller/ll_sw/ull.c
@@ -103,7 +103,7 @@
  *       requiring increased ticker thread context operation queue count.
  */
 #define TICKER_USER_THREAD_OPS   (BT_CTLR_ADV_SET + BT_CTLR_SCAN_SET + \
-				  BT_CTLR_CONN_MAX + \
+				  BT_CTLR_MAX_CONN + \
 				  TICKER_USER_THREAD_VENDOR_OPS + 1)
 #else /* !CONFIG_BT_CTLR_LOW_LAT */
 #define TICKER_USER_LLL_OPS      (2 + TICKER_USER_LLL_VENDOR_OPS + 1)

